src/main/config/config.c

```diff
@@ -201,6 +202,13 @@ static void validateAndFixRatesSettings(void)
     }
 }
 
+static void validateAndFixPositionConfig(void)
+{
+    if (positionConfig()->altNumSatsBaroFallback >= positionConfig()->altNumSatsGpsUse) {
+        PG_RESET(positionConfig);
+    }
```

Comment:
Sorry for getting again with this... resetting all the values maybe is too dangerous. In a future there can be more than now. Is better to reset only "the affected" values.

Reply:
How do I know the default values?

Reply:
Could you please write the exact changes, I expect that if I make some define somewhere - that would be again not what you mean.

Reply:
Fixed, but I guess there will be something wrong with defines..